packages/commerce-sdk-react/src/components/StorefrontPreview/storefront-preview.test.tsx

```diff
@@ -11,6 +11,7 @@ import {detectStorefrontPreview} from './utils'
 import {Helmet} from 'react-helmet'
 import {mockQueryEndpoint, renderWithProviders} from '../../test-utils'
 import {useCommerceApi} from '../../hooks'
+import useConfig from '../../hooks/useConfig'
```

Comment:
I'm wondering if we should change the `index` file to re-export the `useConfig` hook so it's consistent with what we are doing with this "other" hooks.  Something like:

```
// index.ts
export {default as useConfig} from './useConfig'
```

Then we can do this to save an import:

`import {useCommerceApi, useConfig} from '../../hooks'`

Thoughts?